api_docs/markdowns/api/collections/collections.md

```diff
@@ -0,0 +1,227 @@
+# Collections API -- Getting Started [Alpha Version]
+
+## Summary
+
+The goal of this API is to give the possibility to cross data from different sources and different periods on a given project. These data sources are named "collections" in this context. They represent data coming from Botify Analytics, Botify Log Analyzer, and Botify Keywords.
+
+In the manner of an analytics tool, the collections API is queried by providing a set of dimensions and metrics to aggregate data.
+
+## Data Querying Endpoint
+
+- Operation: [[projectQuery;reference#/Project/projectQuery]]
+- Path: `projects/{username}/{project_slug}/query`
+- HTTP Verb: POST
+- Response: `Pagination<BQL_COLLECTIONS_RESULT>`
+
+Body:
+```JSON
+{
+  "collections": [COLLECTIONS],
+  "periods": [PERIODS],
+  "query": BQL_COLLECTIONS_QUERY
+}
+```
+
+### Collections
+
+A Botify Project has multiple collections usable for querying. To retrieve the available collections one can query the endpoint:
+
+- Operation: [[getProjectCollections;reference#/Collections/getProjectCollections]]
+- Path: `projects/{username}/{project_slug}/collections`
+- HTTP Verb: GET
+- Response: `List<Collection>`
+
+Three types of collections are to be distinguished:
```

Comment:
Aussi, est-ce que l'on devrait parler de l'aggregation pour les collections timestampées ?

Reply:
Qu'entends-tu par "agrégation des collections timestampées" ?
Que les métriques sont des agrégats, ou le croisement de plusieurs collections ?

Reply:
Je parlais de rajouter un paragraphe sur la function `first_day_of_week` (bref tout ce que l'on fait avec le Period Selector).
Ca peut attendre une autre issue sur la doc, plus tard.

Reply:
Okay, je propose d'attendre une deuxième itération sur cette documentation. J'ai créé ces deux tickets 
https://botify.atlassian.net/browse/DATA-6101 et https://botify.atlassian.net/browse/DATA-6102 pour les prochaines étapes :) N'hésites pas à les compléter 

Si j'ai votre feu vert, je merge cette PR et deploy developers.botify.com

Reply:
👍